tox.ini

```diff
@@ -62,7 +74,13 @@ passenv =
 extras =
     dev
 deps =
-    importlib_metadata
+    # We explicitly reference `gdal` and `pyproj` here
+    # so that their versions can be constrained
+    # with tox's `--force-dep` option.
+    # This allows to us match the correct versions to what
+    # versions of library files are available.
+    gdal
+    pyproj
```

Comment:
Can we install `numpy` before GDAL here? Does tox install deps in order?

Reply:
It seems all these dependencies are installed in one go, not one-by-one. Installing `numpy` before GDAL will not work. We still need `sitepackages = true`